src/pages/styles/icons.tsx

```diff
@@ -83,6 +94,7 @@ const Page = (): JSX.Element => (
     <Paragraph variant="lead">
       <Text variant="lead">{iconsContent.intro}</Text>
     </Paragraph>
+    <Link href="../components/icon/">{iconsContent['iconPageLink.title']}</Link>
```

Comment:
This should probably start with `../../` since it now refers to `styles/components/icon/` which doesn't exist.

Reply:
Fixed now.